react-components/src/util/table/EnhancedTableHead.js

```diff
@@ -0,0 +1,108 @@
+/**
+ *
+ * @author Sriram
+ *
+ **/
+import React from "react";
+import TableCell from "@material-ui/core/TableCell";
+import TableHead from "@material-ui/core/TableHead";
+import TableRow from "@material-ui/core/TableRow";
+import TableSortLabel from "@material-ui/core/TableSortLabel";
+import Checkbox from "@material-ui/core/Checkbox";
+import Tooltip from "@material-ui/core/Tooltip";
+import Color from "../CyVersePalette";
+import PropTypes from "prop-types";
+import build from "../../util/DebugIDUtil";
+
+class EnhancedTableHead extends React.Component {
+    createSortHandler = property => event => {
+        this.props.onRequestSort(event, property);
+    };
+
+    render() {
+        const {onSelectAllClick, order, orderBy, numSelected, rowCount, columnData, selectable} = this.props;
+
+        return (
+            <TableHead>
+                <TableRow>
+                    {selectable && (
+                        <TableCell padding="checkbox"
+                                   style={{backgroundColor: Color.blue, position: "sticky", top: 0}}>
+                            <Checkbox
+                                indeterminate={numSelected > 0 && numSelected < rowCount}
+                                checked={numSelected === rowCount}
+                                onChange={onSelectAllClick}
+                                style={{
+                                    backgroundColor: Color.blue,
+                                    position: "sticky",
+                                    top: 0,
+                                    color: Color.white,
+                                }}
+                            />
+                        </TableCell>
+                    )
+                    }
+                    {columnData.map(column => {
+                        return (
+                            <TableCell
+                                key={column.name}
+                                numeric={column.numeric}
+                                padding={column.disablePadding ? 'none' : 'default'}
+                                sortDirection={orderBy === column.name ? order : false}
+                                style={{
```

Comment:
There are a few different stylings in this file, could they not be moved to a styles file?